pennylane/ops/op_math/symbolicop.py

```diff
@@ -62,7 +64,7 @@ def __copy__(self):
         return copied_op
 
     # pylint: disable=super-init-not-called
-    def __init__(self, base=None, do_queue=True, id=None):
+    def __init__(self, base, do_queue=True, id=None):
```

Comment:
Just wondering about this change? I suppose it never really was an option to not pass `base`? Or do we loose any abstract default functionality by this change?

Reply:
All properties and methods depend on `base`, thus a `SymbolicOp` with `base=None` would be unusable.

@albi3ro any reason why this default value was added?

Reply:
Can't remember.  `base` is definitely a mandatory argument, so we should indeed get rid of `None`.